src/script.js

```diff
@@ -1,3 +1,41 @@
 "use strict";
 
 // TODO - write your code here.
+
+const randomDamage = () => Math.floor(Math.random() * 10) + 1;
+
+const chooseOption = (opt1, opt2) => {
+    const randNum = Math.floor(Math.random() * 2);
+    return randNum === 0 ? opt1 : opt2;
+  };
+
+const attackPlayer = (health) => health - randomDamage();
```

Comment:
Syntax: when using arrow syntax for functions with a single parameter, parenthesis are optional:

```
const attackPlayer = health => health - randomDamage();
```